api/wing/schema.py

```diff
@@ -72,21 +86,30 @@ class Arguments:
 
     @Auth.user_roles('Admin')
     def mutate(self, info, wing_id, **kwargs):
-        validate_empty_fields(**kwargs)
-        get_wing = Wing.get_query(info)
-        result = get_wing.filter(WingModel.state == "active")
-        exact_wing = result.filter(WingModel.id == wing_id).first()
-        if not exact_wing:
-            raise GraphQLError("Wing not found")
-        admin_roles.create_update_delete_wing()
-        update_entity_fields(exact_wing, **kwargs)
-        payload = {
-            'model': WingModel, 'field': 'name', 'value':  kwargs['name']
-        }
-        with SaveContextManager(
-           exact_wing, 'Wing', payload
-        ):
-            return UpdateWing(wing=exact_wing)
+        try:
+            validate_empty_fields(**kwargs)
+            get_wing = Wing.get_query(info)
+            result = get_wing.filter(WingModel.state == "active")
+            exact_wing = result.filter(WingModel.id == wing_id).first()
+            if not exact_wing:
+                raise GraphQLError("Wing not found")
+            admin_roles.create_update_delete_wing()
+            update_entity_fields(exact_wing, **kwargs)
+            payload = {
+                'model': WingModel, 'field': 'name', 'value':  kwargs['name']
+            }
+            query = Wing.get_query(info)
+            result = query.filter(WingModel.state == "active")
```

Comment:
kindly remove this code because its redundant with the presence of the context manager